packages/backend/src/tokens/request.ts

```diff
@@ -554,11 +554,9 @@ ${error.getFullMessage()}`,
         );
       }
       redirectBackToSatelliteUrl.searchParams.append(constants.QueryParameters.ClerkSynced, 'true');
-      const authErrReason = AuthErrorReason.PrimaryRespondsToSyncing;
-      redirectBackToSatelliteUrl.searchParams.append(constants.QueryParameters.HandshakeReason, authErrReason);
```

Comment:
This isn't actually a handshake, we redirect to the sign in URL, so the reason query parameter being included doesn't make sense.